website/screens/components/sidenav/code/SidenavCodePage.tsx

```diff
@@ -22,216 +22,218 @@ const sections = [
           <HeaderDescriptionCell>Description</HeaderDescriptionCell>
         </tr>
         <tr>
-          <td>children: node</td>
+          <td>title: node</td>
           <td></td>
           <td>
-            The area inside the sidenav. The content will be placed inside a
-            stack with a divider between each element.
+            The area assigned to render the title. It is highly recommended to
+            use the sidenav title.
           </td>
         </tr>
         <tr>
-          <td>title: node</td>
+          <td>children: node</td>
           <td></td>
           <td>
-            The area assigned to render the title. It is highly recommended to
-            use the sidenav title.
+            The area inside the sidenav. The content will be placed inside a
+            stack with a divider between each element.
```

Comment:
This is no longer true since it is the section that provides the divider.

Reply:
Same as above.

Reply:
This don't apply since we are restoring the last implementation.